modules/drivers/oracle/test/metabase/driver/oracle_test.clj

```diff
@@ -19,6 +19,7 @@
             [metabase.query-processor-test :as qp.test]
             [metabase.query-processor-test.order-by-test :as qp-test.order-by-test] ; used for one SSL connectivity test
             [metabase.sync :as sync]
+            [metabase.sync.util]
```

Comment:
our ns linter is complaining that this needs to be without brackets, just as `metabase.sync.util`: https://github.com/metabase/metabase/runs/5248573782?check_suite_focus=true